doc/Type/Any.pod6

```diff
@@ -111,7 +111,7 @@ into the newly created Array.
 
 Defined as:
 
-    multi sub    reverse(*@list  --> Seq:D)
+    multi        reverse(*@list  --> Seq:D)
     multi method reverse(List:D: --> Seq:D)
```

Comment:
While the "sub" is redundant, I prefer to have it in there, just because not everybody knows that "multi" on its own implies "sub".

Reply:
See https://github.com/perl6/doc/issues/1189